calculate_age/calculate.py

```diff
@@ -0,0 +1,69 @@
+import time
+from calendar import isleap
+
+# judge the leap year
+def judge_leap_year(year):
+    if isleap(year):
+        return True
+    else:
+        return False
+
+
+# returns the number of days in each month
+def month_days(month, leap_year):
+    if month in [1, 3, 5, 7, 8, 10, 12]:
+        return 31
+    elif month in [4, 6, 9, 11]:
+        return 30
+    elif month == 2 and leap_year:
+        return 29
+    elif month == 2 and (not leap_year):
+        return 28
+
+# this function to convert a valid number to number of month
+def convert_age_to_months(age):
+  if type(age) == bool: 
```

Comment:
Let's use `isinstance()` method in python.
https://www.w3schools.com/python/ref_func_isinstance.asp